$OpenBSD: patch-src_utils_cpp,v 1.3 2014/10/03 10:24:42 dcoppa Exp $

Use the Adwaita icon theme unconditionally: fixes a problem with
musique GUI not having icons when executed outside of a Desktop
Environment.

This fix is identical to the one in www/minitube, which is by
the same author.

--- src/utils.cpp.orig	Fri Oct 11 03:42:11 2013
+++ src/utils.cpp	Mon Nov 25 17:07:58 2013
@@ -24,6 +24,7 @@ $END_LICENSE */
 #endif
 
 QIcon Utils::themeIcon(const QString &name) {
+    QIcon::setThemeName("Adwaita");
     if (QIcon::hasThemeIcon(name))
         return QIcon::fromTheme(name);
     else
